pixeltable/utils/pytorch.py

```diff
@@ -47,24 +39,7 @@ def __init__(
         with column_type_path.open() as f:
             column_types = json.load(f)
         self.column_types = {k: ColumnType.from_dict(v) for k, v in column_types.items()}
-        self.part_metadata = get_part_metadata(self.path)
-        self._totals = _cumsum([x[1] for x in self.part_metadata])
-
-    def _get_start_position(self, row_start: int) -> Tuple[int, int]:
-        """
-        Returns the starting parquet file and row within that file for a given 'global' row number.
-        based on the individual sizes of each part
-        """
-        assert row_start >= self._totals[0]
-        assert row_start < self._totals[-1]
-
-        prev_acc = 0
-        for i, acc in enumerate(self._totals[1:], start=1):
-            if acc > row_start:
-                return (i - 1, row_start - prev_acc)
-            prev_acc = acc
-
-        assert False, "unreachable"
+        self.part_metadata = sorted(pa.parquet.ParquetDataset(path).files)
```

Comment:
why sorted?

Reply:
different workers may read the same file list in parallel and use it to make allocation decisions, just guaranteeing list order is consistent across these readers. come think of it, this piece of code is run once and then data is pickled is it may not be necessary, only the iteration methods are run parallel, I can remove it.